printer/downloadFile.js

```diff
@@ -0,0 +1,28 @@
+const axios = require('axios');
+const fs = require('fs');
+
+/**
+ * Download PDF file from URL.
+ * @param {Sring} url : PDF file url
+ * @returns filePath: the path to downloaded PDF file
+ */
+async function downloadFileFromURL(url) {
+  await axios({
+    url: url,
+    method: 'GET',
+    responseType: 'stream'
+  })
+    .then(response => {
+      const fileName = 
+        Math.random().toString(36).substring(2, 15) + Math.random().toString(23).substring(2, 5);
+      const filePath = `/tmp/${fileName}.pdf`;
+      fs.writeFileSync(filePath, '');
+      response.data.pipe(fs.createWriteStream(filePath));
+      return filePath;
+    })
+    .catch(error => {
+      console.log(error);
+    });
+}
```

Comment:
add logger.info and logger.error here, if it worked log where the file was downloaded to, if it didnt work log the error